pkg/gadgettracermanager/pubsub/pubsub.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pubsub
+
+import (
+	"sync"
+	"time"
+
+	pb "github.com/kinvolk/inspektor-gadget/pkg/gadgettracermanager/api"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type PubSubEvent struct {
+	Deleted   bool
```

Comment:
I'm just wondering if this should be an enum to make it extensible in case we want to handle different kind of events in the future?

Reply:
I can do that.